src/pages/development/backward-incompatible-changes/reference.md

```diff
@@ -25,6 +25,26 @@ To view changes in functional tests, refer to [Backward incompatible changes in
 
 Patch releases are primarily focused on delivering security and quality enhancements on a regular basis to help you keep your sites performing at their peak. On an exceptional basis, breaking changes or additional patches or hotfixes may be released to address security or compliance issues and high-impact quality issues. On the module level, these are mostly PATCH-level changes; sometimes MINOR-level changes. See [Release policy](https://experienceleague.adobe.com/docs/commerce-operations/release/policy.html).
 
+## 2.4.5 - 2.4.6-beta1
+
+### Magento Open Source
+
+import Docs23 from '/src/pages/_includes/backward-incompatible-changes/open-source/2.4.5-2.4.6-beta1.md'
```

Comment:
For future reference, we should give include directives more identifiable names. I'm the one the used `Docs` originally, but it was a mistake. It should be specific to the include, like `Bic246` or something.

Reply:
I was thinking about it too